services/authorize/client.go

```diff
@@ -2,13 +2,15 @@ package authorize
 
 import (
 	"context"
+	"google.golang.org/grpc/codes"
```

Comment:
Move this import together with the other 3pp imports.

Reply:
fixing